openapi_codec/__init__.py

```diff
@@ -122,7 +122,7 @@ def _parse_document(data, base_url=None):
 
 
 class OpenAPICodec(BaseCodec):
-    media_type = "application/json"
+    media_type = "application/openapi+json"
```

Comment:
+1. Seemed like this was the consensus coming out of: https://github.com/OAI/OpenAPI-Specification/issues/110